modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java

```diff
@@ -118,4 +119,14 @@ public class ServiceOrderEntity implements Serializable {
     @Convert(converter = ObjectJsonConverter.class)
     private Map<String, Object> newConfigRequest;
 
+    @Column(name = "PREVIOUS_LOCK_CONFIG", columnDefinition = "json")
```

Comment:
i would avoid this. Because the columns on this table will be then growing with every new feature added. Instead, can we simply add a new plain JSON  'REQUEST_BODY' and the request body to it. Do you think this would work?

If yes, then we must enter the data for deploy, modifiy, migrate, lock, etc